app/services/catkey.rb

```diff
@@ -21,17 +18,17 @@ def initialize(model)
   # @return [Array<Hash>] a list of catalog links
   def serialize(new_catkeys)
     removed_links = symphony_links - new_catkeys
-    links = (previous_links + removed_links).map { |record_id| { catalog: PREVIOUS_CATKEY, catalogRecordId: record_id, refresh: false } }.uniq
+    links = (previous_links + removed_links).map { |record_id| { catalog: Constants::PREVIOUS_CATKEY, catalogRecordId: record_id, refresh: false } }.uniq
 
-    links + new_catkeys.map.with_index { |record_id, index| { catalog: SYMPHONY, catalogRecordId: record_id, refresh: index.zero? } }
+    links + new_catkeys.map.with_index { |record_id, index| { catalog: Constants::SYMPHONY, catalogRecordId: record_id, refresh: index.zero? } }
   end
 
   def symphony_links
-    find(SYMPHONY)
+    find(Constants::SYMPHONY)
   end
 
   def previous_links
-    find(PREVIOUS_CATKEY)
+    find(Constants::PREVIOUS_CATKEY)
   end
 
   def find(type)
```

Comment:
Note: this class is no longer used for individual catkey editing as changed in this PR, but it is still being used in the change set persister service classes, so I am leaving as is. 

see https://github.com/sul-dlss/argo/blob/main/app/services/collection_change_set_persister.rb#L61 and https://github.com/sul-dlss/argo/blob/main/app/services/item_change_set_persister.rb#L96